### PR TITLE
fix: display initially quoted est. time

### PR DESCRIPTION
**Pull request by dohaki**
This fixes an issue where in-flight deposits close to the short-delay threshold, would cause an estimated time update in the UI. See here https://risklabs.slack.com/archives/C03HV8CFL80/p1719565303568349

We fix this by retrieving the estimated fill time based on the quoted limits. Previously the limits value would update which caused the bug.

### CODE REVIEW

File: src/utils/bridge.ts [hunk 1]
```diff
@@ -106,8 +106,8 @@ export type ConfirmationDepositTimeType = {
 export const getConfirmationDepositTime = (
   amount: BigNumber,
   limits: BridgeLimitInterface,
-  toChain: ChainId,
   fromChain: ChainId,
+  toChain: ChainId,
```
Comment (james-a-morris) [inline src/utils/bridge.ts:110]: Did we mean to change this direction?

**Review by james-a-morris — COMMENTED** [review]
Can you add context to this?

File: src/utils/bridge.ts (hunk 1 above)
Comment (dohaki) [reply src/utils/bridge.ts:110]: Yea, I got confused by the order of the params and refactored it.

**Comment by dohaki** [comment]
> Can you add context to this?

Sorry about that. Added a description

**Review by james-a-morris — APPROVED** [review]

**Pull request merged**